utils/styleguide/styleguide.base.config.js

```diff
@@ -40,14 +40,46 @@ const defaultStyleguideConfig = {
         {
           async: '',
           src: `//www.googletagmanager.com/gtag/js?id=${googleTrackingId}`
+        },
+        {
+          async: '',
+          src: '//static-staging.zdassets.com/customer_analytics_integration/garden_dev/cai.min.js'
```

Comment:
Can we just do straight up https?

Reply:
Just staying consistent w the other protocol-less URL.

Reply:
I think we should just make them all https no need to ever load this over http.